scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2138,6 +2149,12 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
           }
         }
         simpleExprRest(arguments, canApply = true, startPos = startPos)
+      case _: Colon if allowFewerBraces =>
```

Comment:
last question: do we need to check for `canApply` here (and is there a test which confirms that)?

Reply:
Added, it shouldn't break, but it should always be in an apply.